python/cudf_polars/cudf_polars/dsl/expressions/binaryop.py

```diff
@@ -97,14 +97,18 @@ def do_evaluate(
         )
         lop = left.obj
         rop = right.obj
+        print(left, right)
+        print(lop, rop)
         if left.obj.size() != right.obj.size():
             if left.is_scalar:
                 lop = left.obj_scalar
             elif right.is_scalar:
                 rop = right.obj_scalar
-        return Column(
+        res = Column(
             plc.binaryop.binary_operation(lop, rop, self.op, self.dtype),
         )
+        print(res)
+        return res
```

Comment:
I'm guessing all this debugging code can be reverted?

Reply:
Blah. Yep, thanks